src/main/java/seedu/address/model/event/fields/Recurrence.java

```diff
@@ -53,9 +53,10 @@ public Recurrence(String interval) {
     /**
      * Returns if a given string is a valid Recurrence.
      */
-    public static boolean isValidInterval(String trimmedInterval) {
+    public static boolean isValidRecurrence(String trimmedInterval) {
         String lowerCaseInterval = trimmedInterval.toLowerCase(Locale.ROOT);
-        return Objects.equals(lowerCaseInterval, DAILY_CASE)
+        return Objects.equals(lowerCaseInterval, NONE_CASE)
+                || Objects.equals(lowerCaseInterval, DAILY_CASE)
                 || Objects.equals(lowerCaseInterval, WEEKLY_CASE)
                 || Objects.equals(lowerCaseInterval, MONTHLY_CASE) || Objects.equals(lowerCaseInterval, YEARLY_CASE);
```

Comment:
Could be effectively replaced with:

```suggestion
        return List.of(NONE_CASE, DAILY_CASE, WEEKLY_CASE, MONTHLY_CASE, YEARLY_CASE)
                .contains(lowerCaseInterval);
```

Reply:
stonks.